src/frontend/src/common/components/library/data_subview/components/UsherTreeFlow/components/UsherPreparingModal/index.tsx

```diff
@@ -18,32 +20,53 @@ const UsherPreparingModal = ({
   onClose,
 }: Props): JSX.Element => {
   const [hasOpenedUrl, setHasOpenedUrl] = useState<boolean>(false);
+  const [timer, setTimer] = useState<TimeoutType>();
+
+  const openUsher = useCallback(() => {
+    if (!fastaUrl) return;
+
+    const link = document.createElement("a");
+    link.href = `${USHER_URL + fastaUrl}`;
+    link.target = "_blank";
+    link.rel = "noopener";
+    link.click();
+    link.remove();
+
+    onClose();
+    setTimer(undefined);
```

Comment:
Nit: for clarity, I think 36 and 37 should be flip-flopped in order, just so it can't be misread as timer being set to undefined _then_ cleared